pkg/process/procutil/common.go

```diff
@@ -0,0 +1,44 @@
+package procutil
+
+import (
+	"os"
+	"path/filepath"
+)
+
+// HostProc returns the procfs location
+func HostProc(combineWith ...string) string {
+	return GetEnv("HOST_PROC", "/proc", combineWith...)
+}
+
+//GetEnv retrieves the environment variable key. If it does not exist it returns the default.
+func GetEnv(key string, fallback string, combineWith ...string) string {
+	value := os.Getenv(key)
+	if value == "" {
+		value = fallback
+	}
+
+	switch len(combineWith) {
+	case 0:
+		return value
+	case 1:
+		return filepath.Join(value, combineWith[0])
+	default:
+		all := make([]string, len(combineWith)+1)
+		all[0] = value
+		copy(all[1:], combineWith)
+		return filepath.Join(all...)
+	}
+}
```

Comment:
Can we double check that these don't already exist in this repo? I'm pretty sure theres a definition of HostProc somewhere already - perhaps we can kill this whole file

Reply:
This indeed exists, I removed this and used our own function.